### ISSUE TITLE
Are you still maintaining this package?

### ISSUE THREAD

**Issue by Ch4s3** [handnot2/samly#59]
@handnot2 are you still maintaining this package or are you interested in handing it off?

**Comment by k-cross** [comment]
I just forked this under the dropbox repository and am in the process of getting ownership of this through hex.pm.

For both samly and esaml.

**Comment by k-cross** [comment]
@Ch4s3  for reference. I tried contacting in multiple ways in October. If you have some PR suggestions, try creating them there. I'm in the process of doing all the corporate stuff to manage the takeover.


https://github.com/dropbox/esaml

https://github.com/dropbox/samly

**Comment by alexandrubagu** [comment]
@k-cross  any updates ?

**Comment by k-cross** [comment]
@alexandrubagu I'm confused by that statement, need more context. There have been updates to both `esaml` and `samly` yes.

**Comment by alexandrubagu** [comment]
I'm a bit confused, I have a question and I don't know where to ask. The hex.pm package is pointing to [dropbox/samly](https://github.com/dropbox/samly) and seems to be a fork of this one. Can you please provide more details ?

Thanks,

**Comment by k-cross** [comment]
Yes, this repository has been dormant for a long time so I forked the repos to provide maintenance updates and a way for the community to add features they'd like within reason. The goal is to make sure that security and language updates don't break the libraries for people. If you'd like to contribute or call anything out about the libraries please contribute to the dropbox hosted ones (PRs only as issues are not a feature I can turn on and I do not control under corp. policy) instead as those are maintained and validated (manually at this point).

I hope that answers your questions!

**Comment by alexandrubagu** [comment]
@k-cross thanks a lot
